addons/l10n_fr_certification/models/account_closure.py

```diff
@@ -0,0 +1,153 @@
+from datetime import datetime, timedelta
+
+from openerp import models, api, fields
+from openerp.fields import Datetime as FieldDateTime
+from openerp.tools.translate import _
+from openerp.exceptions import UserError
+
+WRITE_MSG = _('Sale Closings are not meant to be written or deleted under any circumstances.')
+
+
+class AccountClosure(models.Model):
```

Comment:
As I don't see any check functions, I assume that the whole purpose is to manually check that the cumulative total of a closure is effectively the period_total + the cumulative of previous period. Is it correct? 
If yes, I think it would be worth storing on the  account.sale.closure, a field to store the cumulative total of previous period: it will allow to compute a boolean to highlight in the list view closures that don't match anymore that computation